### ISSUE TITLE
"default_dashboard" setting

### ISSUE THREAD

**Issue by Geolim4** [marcin-jozwikowski/easy-admin-pretty-urls#37]
Hello Marcin 😂

I have another request for you:

I recently developed a new area of my easy admin project:

A `/volunteer` area for people with very low access and few dedicated controllers.

Everything is working well except one thing:

```yml
pretty_routes_admin:
    resource: 'src/Controller/Admin'
    type: 'pretty_routes'
    prefix: /admin

pretty_routes_volunteer:
    resource: 'src/Controller/Volunteer'
    type: 'pretty_routes'
    prefix: /espace-benevole
```

The routes are well rewritten, except I'm getting the Dashboard menu from my Admin area due to this settings:
`default_dashboard`

I think that you should give the (optional) ability to set a custom dasboard inside the `PrettyRoutesController` attribute, something like:

```php
#[PrettyRoutesController(path: 'candidats', dasboard: YourDashboardClass:class)]
```

If the **dasboard** parameter is set then your code use it, then it fallback to the "default_dashboard" settings. 

What do you think of it ?

**Comment by Geolim4** [comment]
I think that the magic stuff could be done in `\MarcinJozwikowski\EasyAdminPrettyUrls\Service\ClassAnalyzer::makeRouteDto`:


Something like this
```php
    private function makeRouteDto(ReflectionClass $reflection, string $action, string $actionPath): ActionRouteDto
    {
        // determine the first part of the final URL - classname or the value from PrettyRoutesController attribute
        $classAttribute = $this->getControllerAttribute($reflection);
        $routePathFormat = '/%s/%s';
        $routeDefaults = [
            // set route values
            '_controller' => $classAttribute?->getArguments()[PrettyRoutesController::ARGUMENT_DASHBOARD] ?? $this->prettyUrlsDefaultDashboard,
            PrettyUrlsGenerator::EA_FQCN => $reflection->getName(),
            PrettyUrlsGenerator::EA_ACTION => $action,
        ];

        if ($this->prettyUrlsIncludeMenuIndex) {
            // add menu properties when required
            $routePathFormat .= '/{menuPath}';
            $routeDefaults[PrettyUrlsGenerator::MENU_PATH] = '-1,-1';
        }

        $simpleName = $this->routeNamingGenerator->generateSimplifiedClassName($reflection->getName());
        $classPath = $classAttribute?->getArguments()[PrettyRoutesController::ARGUMENT_PATH] ?? $simpleName;

        return new ActionRouteDto(
            name: $this->routeNamingGenerator->generateRouteName($simpleName, $action),
            path: sprintf($routePathFormat, $classPath, $actionPath),
            defaults: $routeDefaults,
        );
    }
```

A small update of the `PrettyRoutesController` class to add new attribute:

```php
class PrettyRoutesController
{
    public const ARGUMENT_ACTIONS = 'actions';
    public const ARGUMENT_CUSTOM_ACTIONS = 'customActions';
    public const ARGUMENT_PATH = 'path';
    public const ARGUMENT_DASHBOARD = 'dashboard';
    public const DEFAULT_ACTIONS = [
        Action::INDEX,
        Action::NEW,
        Action::DETAIL,
        Action::EDIT,
        Action::DELETE,
        Action::BATCH_DELETE,
        'renderFilters',
        'autocomplete',
    ];

    /**
     * @param string[]|null $actions
     * @param string[]|null $customActions
     */
    public function __construct(
        public ?array $actions = null,
        public ?array $customActions = null,
        public ?string $path = null,
        public ?string $dashboard = null,
    ) {
    }
}
```


With a new attribute set on your controller:
```php
#[PrettyRoutesController(path: 'candidats', dashboard: '\App\Controller\Volunteer\DashboardController::index')]
```

This approach works for me, but feel free to improve it :D

**Comment by Geolim4** [comment]
Have aa nice PR @marcin-jozwikowski  😇

**Comment by marcin-jozwikowski** [comment]
@Geolim4 Thanks again for implementing this! I've already released it as v2.8.0.

**Comment by Geolim4** [comment]
Great, thanks you Marcin it will help me a lot !
